Makefile

```diff
@@ -427,3 +426,7 @@ check_abci_specs:
 	python -m autonomy.cli analyse abci generate-app-specs packages.valory.skills.transaction_settlement_abci.rounds.TransactionSubmissionAbciApp packages/valory/skills/transaction_settlement_abci/fsm_specification.yaml || (echo "Failed to check transaction_settlement_abci consistency" && exit 1)
 	echo "Successfully validated abcis!"
 
+
+AEA_AGENT_HASH:="valory/hello_world:latest:"$(shell cat packages/hashes.csv | grep "agents/hello_world" | cut -d "," -f2 )
```

Comment:
@8ball030 now it works

Reply:
NIce nice nice Thats a lovely one liner.